apps/services/search-indexer/src/migrate/lib/elastic.ts

```diff
@@ -165,3 +169,92 @@ export const removeIndexIfExists = async (indexName: string) => {
 
   return false
 }
+
+/**
+ * Read all documents from the old index and write their popularity scores to
+ * the documents with corresponding ids in the new index
+ *
+ * @param {string} oldIndex
+ * @param {string} newIndex
+ */
+export const migratePopularityScores = async (
+  oldIndex: string,
+  newIndex: string,
+) => {
+  const query = {
+    query: {
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      match_all: {},
+    },
+    size: 10000,
```

Comment:
I added a scroll query so that the state of the index is preserved while we are scrolling through. If we ever upgrade ES to >=7.10 then we can use a point in time instead.